examples/template-svelte-js/rollup.config.js

```diff
@@ -0,0 +1,100 @@
+import svelte from 'rollup-plugin-svelte';
```

Comment:
I'm new to Svelte; is this `rollup.config.js` file necessary for running Svelte apps?

Reply:
Svelte is a language rather than framework, there is Svelte compiler which compiles `.svelte` files to JavaScript.
Some kind of build tool is needed to run the Svelte compiler during development and build.

But for actually running the Svelte app in the browser there is no need for Rollup.

Reply:
Sounds good, let's try to keep only the files that are needed for running the app in-browser.

Really appreciate this PR!~

Reply:
No problem, I am aware of that it needs to be closer to the existing ones.

I think I did not explain well the importance of Rollup:

I have created this template based on the official Svelte template: https://github.com/sveltejs/template 
That template provides a `rollup.config.js` which is needed to be able to run `npm run dev` or `npm run build`. Removing the config would mean that the user has to create it first, and then they can develop or build their app.

Reply:
Makes sense, we can keep it then.